### ISSUE TITLE
web.whatsapp.com

### ISSUE THREAD

**Issue by vdias** [StevenBlack/hosts#798]
web.whatsapp.com is being wrongly blocked.

![2018-10-08 09_03_58-mozilla firefox](https://user-images.githubusercontent.com/2320553/46595588-6110b900-cad9-11e8-840c-fa7cc2d9eda6.png)

**Comment by kronflux** [comment]
That's a message VERY specifically telling you in plain English, that one of your addons, uBlock Origin, is blocking it. NOT this hosts file.
uBlock Origin is a completely unrelated project.
You also have at the bottom of that page to disable the block for that page with that addon.

**Comment by kronflux** [comment]
First off, that message is from uBlock Origin, you have the ability to add an exception for this block.

Secondly, you're using a list (fakenews-social), which specifically blocks social media - which would include WhatsApp. So nothing is "wrongly" being blocked.

**Comment by lightswitch05** [comment]
as kronflux points out, `web.whatsapp.com` and other whatsapp subdomains are in the [social list](https://github.com/Sinfonietta/hostfiles/blob/5efbc2d754435c526df7713ed64789679f78a765/social-hosts#L952). Either whitelist it or do not use the social extension
